src/cdk/overlay/position/global-position-strategy.ts

```diff
@@ -147,10 +149,10 @@ export class GlobalPositionStrategy implements PositionStrategy {
     const element = this._overlayRef.overlayElement;
 
     if (!this._wrapper && element.parentNode) {
-      this._wrapper = document.createElement('div');
-      this._wrapper.classList.add('cdk-global-overlay-wrapper');
-      element.parentNode.insertBefore(this._wrapper, element);
-      this._wrapper.appendChild(element);
+      this._wrapper = this._document.createElement('div');
+      this._wrapper!.classList.add('cdk-global-overlay-wrapper');
```

Comment:
Put the `!` on the assignment for `_wrapper`?
```ts
this._wrapper = this._document.createElement('div')!;
```

Reply:
TS can't figure it out when assigning to a property.